sycl/include/CL/sycl/detail/image_impl.hpp

```diff
@@ -37,18 +38,21 @@ namespace detail {
 using image_allocator = aligned_allocator<byte>;
 
 // utility function: Returns the Number of Channels for a given Order.
-uint8_t getImageNumberChannels(image_channel_order Order);
+SYCL_API uint8_t getImageNumberChannels(image_channel_order Order);
 
 // utility function: Returns the number of bytes per image element
-uint8_t getImageElementSize(uint8_t NumChannels, image_channel_type Type);
+SYCL_API uint8_t getImageElementSize(uint8_t NumChannels,
+                                     image_channel_type Type);
 
-RT::PiMemImageChannelOrder convertChannelOrder(image_channel_order Order);
+SYCL_API RT::PiMemImageChannelOrder
+convertChannelOrder(image_channel_order Order);
 
-image_channel_order convertChannelOrder(RT::PiMemImageChannelOrder Order);
+SYCL_API image_channel_order
+convertChannelOrder(RT::PiMemImageChannelOrder Order);
 
-RT::PiMemImageChannelType convertChannelType(image_channel_type Type);
+SYCL_API RT::PiMemImageChannelType convertChannelType(image_channel_type Type);
```

Comment:
Off-topic. Probably we could move modified functions in this file to source dir.